Modules/SubModule_Population/xASL_wrp_CreatePopulationTemplates.m

```diff
@@ -293,9 +293,13 @@ function xASL_wrp_CreatePopulationTemplates(x, bSaveUnmasked, bCompute4Sets, Spe
 %% 2. Iterate over scan types & sessions
 for iScanType=1:length(PreFixList)
     UnAvailable = 0;
+
+	% Define sessions
+	x.S.InputDataStr = PreFixList{iScanType};
+	[~, ~, listSessions] = xASL_adm_GetPopulationSessions(x);
     
     fprintf('%s\n', ['Searching ' TemplateNameList{iScanType} ' images:']);
-    for iSession=1:length(x.SESSIONS) % iterate over sessions
+    for iSession=1:length(listSessions) % iterate over sessions
```

Comment:
Then we can replace `length(listSessions)` by `nSessions` throughout

Reply:
Done